src/main/java/com/artipie/PermissionsYaml.java

```diff
@@ -36,7 +37,7 @@
  * on repository yaml configuration file.
  * @since 0.2
  */
-public final class RpPermissions implements Permissions {
+public final class PermissionsYaml implements Permissions {
```

Comment:
@g4s8 please consider renaming the class to `YamlPermissions` as it more common to put implementation specifics first and interface name second